lectures/5-Tutorials/SymmetricTensors.md

```diff
@@ -0,0 +1,793 @@
+end
+TT
+```
+Circling back to our earlier remark, we clearly see that the fusion trees of this tensor indeed have non-trivial vertex labels.
+```{code-cell} julia
+f = collect(fusiontrees(TT))[3][2]
+```
+```{code-cell} julia
+f.vertices
+```
+
+```{note}
+While we have given an explicit example using $\mathrm{SU}(3)$ with the adoint irrep on the physical level, the same construction holds for the general $\mathrm{SU}(N)$ with arbitrary physical irreps. All we require is the expression for the eigenvalues of the quadratic Casimir in each irrep.
+```
+
+
+## Level 5: Anyonic Symmetries and the Golden Chain
+
+While we have focussed exclusively on group-like symmetries in our discussion so far, the framework of symmetric actually extends beyond groups to so-called [*categorical symmetries*](https://jutho.github.io/TensorKit.jl/stable/man/sectors/#ss_representationtheory). These are quite exotic symmetries characterized in terms of [the topological data of a unitary fusion category](https://jutho.github.io/TensorKit.jl/stable/man/categories/#ss_topologicalfusion). While the precise details of all the terms in these statements fall beyond the scope of this tutorial, we can give a simple example of a Hamiltonion model with a categorical symmetry called [the golden chain](https://arxiv.org/abs/cond-mat/0612341).
+
+This is a one-dimensional system defined as a spin chain, where each physical 'spin' corresponds to a so-called [Fibonacci anyon](https://arxiv.org/abs/0902.3275). There are two such Fibonacci anyons, which we will denote as $1$ and $\tau$. They obey the fusion rules
+```{math}
+1 \otimes 1 = 1, \quad 1 \otimes \tau = \tau, \quad \tau \otimes \tau = 1 \oplus \tau.
+```
+The Hilbert space of a chain of Fibonacci anyons is not a regular tensor product space, but rather a *constrained Hilbert space* where the only allowed basis states are labeled by valid Fibonacci fusion configurations.
+In the golden chain model, we define a nearest-neighbor Hamiltonian on this Hilbert space by imposing an energy penalty when any two neighboring anyons fuse to a $\tau$ anyon.
+
+Even just writing down an explicit expression for this interaction on such a constrained Hilbert space is not entirely straightforward. However, using the framework of symmetric tensors it can actually be explicitly constructed in a very straightforward way. Indeed, TensorKit.jl supports a dedicated [`FibonacciAnyon`](https://jutho.github.io/TensorKit.jl/latest/lib/sectors/#TensorKit.FibonacciAnyon) sector type which can be used to construct precisely such a constrained Fibonacci-graded vector space. A Hamiltonian
+```{math}
+H = \sum_{\langle i,j \rangle} h_{ij}
+```
+which favors neighboring anyons fusing to the vacuum can be constructed as a tensor map on the product space of two Fibonacci-graded physical spaces
+```{code-cell} julia
+:tags: [hide-output]
+V = Vect[FibonacciAnyon](:τ => 1)
+```
+and assigning the following nonzero block value to the two-site fusion trees
```

Comment:
I think the colon in the figure is a bit strange?

Reply:
It's the same style as every tensor up till then. Is it particularly strange for this one?